src/layouts/Home/components/Foundation.tsx

```diff
@@ -163,8 +119,15 @@ export const Foundation: React.FC = () => {
                   group="Content"
                   title="The principles of language for writing products"
                   url="/content"
-                  image={contentImage}
-                />
+                >
+                  <StaticImage
+                    alt=""
+                    placeholder="tracedSVG"
+                    src="../../../images/home-pillars-content.svg"
+                    tracedSVGOptions={{ color: '#478d6b', background: '#F6F4F4' }}
+                    imgStyle={imgStyles}
+                  />
```

Comment:
Using an `img` tag would remove the Gatsby image's [placeholder](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image/#placeholder) feature. Rather than using static images, I think using the `gatsby-image-plugin` placeholder option of `DOMINANT_COLOR` would be better for the home page pillar images. It would be a three line change (deletion) and keep the change simplified. Also, the aesthetics would more closely map to what's in production in that there is a placeholder for each home page pillar image.